internal/sbi/producer/ue_context.go

```diff
@@ -532,8 +533,10 @@ func RegistrationStatusUpdateProcedure(ueContextID string, ueRegStatusUpdateReqD
 				logger.GmmLog.Errorf("AM Policy Control Delete Error[%v]", err.Error())
 			}
 		}
-
-		gmm_common.RemoveAmfUe(ue, false)
+		// TODO: Currently only consider the 3GPP access type
+		if !ue.UeCmRegistered[models.AccessType__3_GPP_ACCESS] {
```

Comment:
Will 3gpp and non-3gpp have different behavior here? I think a better implementation here is to remove the context according to different access type (which is specified during context transfer procedure), but I understand it seems that currently we can only remove the complete AmfUe context (3gpp+non-3gpp). In this case, I don't quite understand why non-3gpp is excluded here😂